arm_robots/src/arm_robots/base_robot.py

```diff
@@ -21,6 +20,7 @@ def __init__(self, robot_namespace: str = '', robot_description: str = 'robot_de
         TrajectoryFollower class, we do not want BaseRobot to use any MoveIt TrajectoryExeuction or rely on any trajectory
         execution services.
         """
+        robot_description = rospy.resolve_name(robot_description)
```

Comment:
why have this extra line instead of
`self.robot_description = rospy.resolve_name(robot_description)` ?